documentation/modules/managing/con-service-discovery.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// managing/assembly-management-tasks.adoc
+
+[id='proc-add-service-discovery-{context}']
+= Discovering services using labels and annotations
+
+Service discovery makes it easier for client applications running in the same cluster as {ProductName} to interact with a Kafka cluster.
+
+A _service discovery_ label and annotation is generated for services used to access the Kafka cluster:
+
+* Internal Kafka bootstrap service
+* HTTP Bridge service
+
+The label helps find the service, and the annotation provides connection details that a client application can use to make the connection.
```

Comment:
"The label helps find..." is a little bit abstract. How about "makes the service discoverable", "makes the service findable", or similar wording.

Reply:
I guess the service can be discovered without the label, but the label is making it easier. 
How about: _The label helps to make the service discoverable,_

Reply:
Yes, that works for me.